domains/frogiee1.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "used for extra links on my site, wildcard included too.",
+
+    "domain": "is-cool.dev",
+    "subdomain": "*.frogiee1",
```

Comment:
You have the`*` wildcard listed for your subdomain but the file name does not have the wildcard. Can you please give us a reason why you need a wildcard subdomain?

Reply:
as i own an "unblocked games" site, instead of me having to beg for extra subdomains, i could just use a wildcard instead. also can be used as a last resort if frogiee.one expires